spec/services/moves/reference_generator_spec.rb

```diff
@@ -10,8 +10,10 @@
   let(:existing_reference) { '12345678' }
   let!(:move) { create :move, reference: existing_reference }
 
+  # Faker uses random numbers, so this test will need to be changed if
+  # more Faker data is introduced to move (or any of its associations)
   it 'generates a new reference' do
```

Comment:
This doesn't seem very reliable as presumably if a random number is used elsewhere then a different ID will be returned... how about instead redefining `PERMISSIBLE_CHARACTERS = ['A']` and `PERMISSIBLE_NUMBERS=[1]` (with a spy?) so you can guarantee the generated reference ?

Reply:
however, not a bigly issue so approving as-is